ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "compute_kernel_config.hpp"
 
+#define MAX_DEST_REG_COUNT 16
```

Comment:
Does it make sense to use one of the defines already defined in these files ?

./tt_metal/common/constants.hpp
./tt_metal/hw/inc/grayskull/tensix_types.h
./tt_metal/hw/inc/wormhole/wormhole_b0_defines/tensix_types.h
./tt_metal/hw/inc/blackhole/tensix_types.h

Reply:
Would MAX_DST_SIZE in tt_metal/common/constants.hpp be acceptable?

Reply:
Do you want to do something like 
#if defined(GRAYSKULL)
use the one in grayskull/tensix_types.h
#elif defined(WORMHOLE) 
use the one in wormhole/tensix_types.h 


This is to make sure if future architectures change with larger/smaller dst sizes we still have this code working

Reply:
Updated the code to calculate the maximum dest registers.

I was able to get DEST_REGISTER_FULL_SIZE, but DATUMS_PER_ROW is in common/inc/ckernel_defs.h which is too hardware specific to include here.